scripts/lynx/deploy_bqeth.py

```diff
@@ -9,14 +9,19 @@
 from deployment.registry import merge_registries
 
 VERIFY = False
-CONSTRUCTOR_PARAMS_FILEPATH = CONSTRUCTOR_PARAMS_DIR / "lynx" / "beta_program_initiator.yml"
+CONSTRUCTOR_PARAMS_FILEPATH = CONSTRUCTOR_PARAMS_DIR / "lynx" / "bqeth.yml"
 LYNX_REGISTRY = ARTIFACTS_DIR / "lynx.json"
 
 
 def main():
     deployer = Deployer.from_yaml(filepath=CONSTRUCTOR_PARAMS_FILEPATH, verify=VERIFY)
-    beta_program_initiator = deployer.deploy(project.BetaProgramInitiator)
-    deployments = [beta_program_initiator]
+
+    global_allow_list = deployer.deploy(project.GlobalAllowList)
```

Comment:
Should `GlobalAllowList` be deployed separately? Related to comment here - https://github.com/nucypher/nucypher-contracts/pull/280#discussion_r1668816749

Reply:
Yes.